public/content/lessons/2017/derivatives-trig-functions/index.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Trig Derivatives through geometry
+description: Introduction to the derivatives trigonometric functions thought about geometrically and intuitively.
+date: 2017-04-30
+chapter: 4
+video: S0_qX4VJhMQ
+timestamp: 756
+source: _2017/eoc/chapter3.py
+credits:
+- Lesson by Grant Sanderson
+- Text adaptation by Kurt Bruns
+---
+
+Let’s tackle one more type of function: Trigonometric functions. In particular, let’s focus on the sine function. Right here I’m going to assume you are familiar with how to think about trigonometric functions using the unit circle; the circle with radius $1$ centered at the origin.
+
+For a given value of theta, like $0.8$, you’d imagine walking around the circle, starting from the rightmost point, until you’ve traversed that distance $0.8$ in arc length. This is the same thing as saying that this angle here is exactly $0.8$ radians, since the circle has radius $1$. Then $\sin(\theta)$ is your height above the $x$-axis at this point.
+
+<Figure
+  image="figures/13.16.svg"
```

Comment:
Add a small arc and a label theta to emphasize that theta refers to the angle between the red line and the x-axis.